tutorials/neurips_challenge_quickstart.md

```diff
@@ -34,7 +34,7 @@ These don't include models that have been finetuned or otherwise aligned, as per
 | Models in Lit-GPT         | Reference                                                    |
 | ------------------------- | ------------------------------------------------------------ |
 | Meta AI Llama 2 Base      | [Touvron et al. 2023](https://arxiv.org/abs/2307.09288)      |
-| TII UAE Falcon Base       | [TII 2023](https://falconllm.tii.ae/)     
+| TII UAE Falcon Base       | [TII 2023](https://falconllm.tii.ae/)
```

Comment:
This table was missing the rightmost "|"?

GitHub seems to render it correctly anyways

Reply:
Eagle eye 😄.
Yes, indeed, it's missing.
GitHub does a lot of things behind the curtains to help users, and it's great, but I don't like to rely on it too much.
Fixed.